test/Microsoft.AspNetCore.Hosting.Tests/HostingApplicationTests.cs

```diff
@@ -172,7 +170,29 @@ public void ActivityIsStoppedDuringUnhandledExceptionCall()
             }));
 
             var context = hostingApplication.CreateContext(features);
-            hostingApplication.DisposeContext(context, null);
+            hostingApplication.DisposeContext(context, new Exception());
+            Assert.True(endCalled);
+        }
+
+        [Fact]
+        public void ActivityIsAvailableDuringUnhandledExceptionCall()
+        {
+            var diagnosticSource = new DiagnosticListener("DummySource");
+            var hostingApplication = CreateApplication(out var features, diagnosticSource: diagnosticSource);
+
+            bool endCalled = false;
+            diagnosticSource.Subscribe(new CallbackDiagnosticListener(pair =>
+            {
+                if (pair.Key == "Microsoft.AspNetCore.Hosting.UnhandledException")
+                {
+                    endCalled = true;
```

Comment:
In case of exception we expect  Exception event followed by Stop event. 
It seems that test does not check Stop event, is it correct?